tests/zipkin_decorator_test.py

```diff
@@ -59,6 +59,31 @@ def test_client_request_option_decorator():
         )
 
 
+def test_client_explicit_context():
+    trace_id = 'trace_id'
+    span_id = 'span_id'
```

Comment:
Since we're not asserting anything about these (unlike the test above), you don't need to store these in a variable. Just pass in whatever.